### PR TITLE
Media query breakpoints

### PR DESCRIPTION
**Pull request by FalkoJoseph**
Fixed media breakpoints.

### CODE REVIEW

File: _assets/stylesheets/_mixins.scss [hunk 1]
```diff
@@ -7,15 +7,15 @@
 
 @mixin breakpoint($val) {
   @if $val == small {
-    @media ($small) {
+    @media ("max-width: 767px") {
```
Comment (nielsenramon) [inline _assets/stylesheets/_mixins.scss]: Add `all and ("max-wi...)` please.

File: _assets/stylesheets/_mixins.scss [hunk 2]
```diff
@@ -7,15 +7,15 @@
...
       @content;
     }
   } @else if $val == medium {
-    @media ($medium) {
+    @media ("max-width: 1024px") {
```
Comment (nielsenramon) [inline _assets/stylesheets/_mixins.scss]: Add `all and ("max-wi...)` please.

File: _assets/stylesheets/_mixins.scss [hunk 3]
```diff
@@ -7,15 +7,15 @@
...
       @content;
     }
   } @else if $val == large {
-    @media ($large) {
+    @media ("max-width: 1223px") {
```
Comment (nielsenramon) [inline _assets/stylesheets/_mixins.scss]: Add `all and ("max-wi...)` please.

**Comment by nielsenramon** [comment]
LGTM, squash please and I merge it! :star:

**Comment by FalkoJoseph** [comment]
@nielsenramon :package:

**Comment by nielsenramon** [comment]
Can you rebase with `f` parameter so we do not have the old commit messages in there + name your commit `Update media queries`? :police_car:

**Pull request merged**